.github/workflows/shippable_builds.yml

```diff
@@ -697,11 +740,14 @@ jobs:
           APPLICATION_ID: ${{ steps.pkginfo.outputs.APPLICATION_ID }}
           REPO: ${{ github.repository }}
           APP_SHA: ${{ steps.shanotes.outputs.app_sha }}
+          RELEASE_TYPE: ${{ vars.RELEASE_TYPE }}
         run: |
           # r0adkll/upload-google-play expects the release notes in a different structure
-          FILEPATH=app-metadata/${APPLICATION_ID}/en-US/changelogs/${VERSION_CODE}.txt
           mkdir whatsnew
-          wget -O whatsnew/whatsnew-en-US https://raw.githubusercontent.com/${REPO}/${APP_SHA}/${FILEPATH}
+          if [[ "$RELEASE_TYPE" = "beta" || "$RELEASE_TYPE" = "release" ]]; then
```

Comment:
For daily we should have some release notes that indicate the sha it was built from, and a direct link to the workflow run. Any other info you think would be relevant? 

While you are here, can you add `?token=<timestamp here>` to the end of the URL? Otherwise github may do some caching as I noticed in render-notes